open_academy/views/course_view_form.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="course_view_form"  model="ir.ui.view">
+        <field name="name">Form</field>
```

Comment:
the name of views with type form, tree, search, etc follows the conventions as the xml id please name course.view.form and fix the name for the other views